### PR TITLE
Fix 'Most installed version' on port installation stats

### LINKED ISSUE #123: Incorrect ‘most installed version’ is reported.
Each port seems to give ‘user(s)’ as the most installed version.... see e.g.

https://ports.macports.org/port/clang-8.0/stats?days=30&days_ago=0

### PR DESCRIPTION
**Pull request by arjunsalyan**
Closes: https://github.com/macports/macports-webapp/issues/123

### CODE REVIEW

**Comment by arjunsalyan** [comment]
I have tested this and it is a minor change, so I am merging this PR as the functionality is broken.

File: app/ports/templates/ports/port-detail/installation_stats.html [hunk 1]
```diff
@@ -37,7 +37,7 @@ <h4 class="border-bottom">Statistics: <span style="font-size:12px"><button type=
                 </tr>
                 <tr>
                     <th scope="row">Most Installed Version</th>
-                    <td><strong>{{ port_installations_by_port_version.first.version }}</strong> : {{ port_installations_by_port_version.first.num }} user(s)</td>
```
Comment (umeshksingla) [inline app/ports/templates/ports/port-detail/installation_stats.html:40]: Why did this stop working in the first place?

Comment (arjunsalyan) [reply app/ports/templates/ports/port-detail/installation_stats.html:40]: There were changes done to the `port_installations_by_port_version` query set:
1. Versions are no longer sorted by number of users, so first object will not give the most installed version.
2. It is no longer a query set, but a list of dictionaries.

**Comment by mojca** [comment]
Thanks for fixing this.

But the sole fact that this got broken without anyone noticing is a strong suggestion that we need an unit test!!!

**Comment by umeshksingla** [comment]
I still don't understand why this broke in the first place. I tried going through the commits where it could have last changed but couldn't find anything. Maybe I need to take a deeper look.

**Comment by mojca** [comment]
#116?

**Comment by umeshksingla** [comment]
Oh. I skipped that latest commit because I felt why would `port_installations_by_port_version` be affected by something related to visual ordering ever. Turns out the commit was made to fix something else which in turn fixed x-axis sorting by itself.

This was stupid of me.

**Pull request merged**